src/locales/en/translations.json

```
@@ -76,7 +76,7 @@
     "IN_COMMERCIAL_REG_NUMBER": "Please enter a valid number. Hint: Alphanumeric with hyphen with 4 to 21 digits.",
     "IN_VAT_ID": "Please enter a valid number. Hint: Alphanumeric with hyphen with 5 to 15 digits.",
     "IN_LEI_CODE": "Please enter a valid number. Hint: Alphanumeric with exact 20 digits.",
-    "DE_COMMERCIAL_REG_NUMBER": "Please enter a valid number. Hint: Alphanumeric with hyphen and space.",
+    "DE_COMMERCIAL_REG_NUMBER": "Please enter a valid identifier. Hint: Starting with court name or code and followed by registered identifier.",
```

As discussed yesterday, it was also recommended to add examples.
```suggestion
    "DE_COMMERCIAL_REG_NUMBER": "Please enter a valid identifier. Hint: Starting with court name or code and followed by registered identifier. Examples: HRB 98814, München HRB 175450, F1103R_HRB98814.",
```